_posts/2017-10-17-cockpit-153.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Cockpit 153
+author: pitti
+date: 2017-10-17 10:00
+tags: cockpit linux
+slug: cockpit-153
+category: release
+summary: Cockpit with new oVirt package
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release regularly.
+Here are the release notes from version 153.
+
+### Add oVirt package
+
+This version introduces the "oVirt Machines" page in the new cockpit-ovirt
+package on Fedora for controlling oVirt virtual machine clusters.
+This code was moved into Cockpit as it shares a lot of code with the existing
+"Machines" page, which manages virtual machines through libvirt.
+
+Note that when installed, it will hide the "Machines" page.
```

Comment:
Lets use the actual package name here: "This feature is packaged in ```cockpit-ovirt``` and when installed it will replace the "Machines" page."

Reply:
Thanks, updated. I took out the package name from the first paragraph in exchange, to avoid repetition.